README.md

```diff
@@ -0,0 +1,169 @@
+	//
+	// We don't provide a list of supported ciphers. This is what OpenSSL
+	// does. The only limitation is usage of TLSv1.2 (because other protocol
+	// versions don't seem to support the GOST cipher). To add additional
+	// ciphers (GOST cipher), you must configure OpenSSL.
+	//
+	// See also
+	//
+	// * https://www.openssl.org/docs/man1.1.1/man1/ciphers.html
+	SslCiphers string
+	// SslPassword is a password for decrypting the private SSL key file.
+	// The priority is as follows: try to decrypt with SslPassword, then
+	// try SslPasswordFile.
+	SslPassword string
+	// SslPasswordFile is a path to the list of passwords for decrypting
+	// the private SSL key file. The connection tries every line from the
+	// file as a password.
+	SslPasswordFile string
+}
+```
+To create a connection from the created dialer a `Dial` function could be used:
+```go
+package tarantool
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/tarantool/go-tarantool/v2"
+	"github.com/tarantool/go-tlsdialer"
+)
+
+func main() {
+	dialer := tlsdialer.OpenSslDialer{
+		Address: "127.0.0.1:3301",
+		User:    "guest",
+	}
+	opts := tarantool.Opts{
+		Timeout: 5 * time.Second,
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 500*time.Millisecond)
+	defer cancel()
+
+	conn, err := tarantool.Connect(ctx, dialer, opts)
+	if err != nil {
+		fmt.Printf("Failed to create an example connection: %s", err)
+		return
+	}
+
+	// Use the connection.
+	data, err := conn.Do(tarantool.NewInsertRequest(999).
+		Tuple([]interface{}{99999, "BB"}),
+	).Get()
+	if err != nil {
+		fmt.Printf("Error: %s", err)
+	} else {
+		fmt.Printf("Data: %v", data)
+	}
+}
+```
+
+## Application build
+
+Since tlsdialer uses SSL for connection to the Tarantool-EE, Cgo should be
+enabled while building and OpenSSL libraries and includes should be available
+in build time.
+
+### Building with system OpenSSL
+
+Build your application using the command:
+1. **Static build**.
+   ```shell
+   CGO_ENABLED=1 go build -ldflags "-linkmode external -extldflags '-static -lssl -lcrypto'" -o myapp main.go
+   ```
+2. **Dynamic build**.
+   ```shell
+   CGO_ENABLED=1 go build -o myapp main.go
+   ```
+
+### Building with a custom OpenSSL version
+
+OpenSSL could be build in two ways. Both of them require downloading the source
+code of OpenSSL. It could be done from the [official website](https://www.openssl.org/source/)
+or from the [GitHub repository](https://github.com/openssl/openssl).
+1. **Static build**. Run this command from the installation directory to configure
+   the OpenSSL:
+   ```shell
+   ./config no-shared --prefix=/tmp/openssl/ --openssldir=/usr/local/ssl/
+   ```
+2. **Dynamic build**. Run this command from the installation directory to configure
+   the OpenSSL:
+   ```shell
+   ./config --prefix=/tmp/openssl/ --openssldir=/usr/local/ssl/
+   ```
+   After configuring, run this command to install and build OpenSSL:
+   ```shell
+   make install --prefix=/tmp/openssl/
+   ```
```

Comment:
It will install to the prefix anyway.
```suggestion
   ```shell
   make install
   ```
```

Reply:
Updated.